demonstrations/HHL_with_catalyst_qrisp.metadata.json

```
@@ -0,0 +1,48 @@
+{
+    "title": "HHL with Qrisp x Catalyst",
+    "authors": [
+        {
+            "username": "renezander"
+        },
+        {
+            "username": "positr0nium"
+        },
+        {
+            "username": "matic_p"
+        }
+    ],
+    "dateOfPublication": "2025-02-11T09:00:00+00:00",
+    "dateOfLastModification": "2025-02-11T09:00:00+00:00",
+    "categories": [
+        "How-to",
+        "Algorithms"
+    ],
+    "tags": [],
+    "previewImages": [
+        {
+            "type": "thumbnail",
+            "uri": "/_static/demo_thumbnails/regular_demo_thumbnails/thumbnail_HHL_with_catalyst_qrisp.png"
+        },
+        {
+            "type": "large_thumbnail",
+            "uri": "/_static/demo_thumbnails/large_demo_thumbnails/thumbnail_large_HHL_with_catalyst_qrisp.png"
+        }
+    ],
+    "seoDescription": "SEO description to be filled in (TBD)",
```

@alvaro-at-xanadu - Let me know if you have any suggestions for SEO description.

```suggestion
    "seoDescription": "Learn how to implement the HHL quantum algorithm for solving linear systems using Qrisp and Catalyst.",
```

---

I think this looks good :)